packages/animations/README.md

```diff
@@ -9,10 +9,13 @@ cd example/
 flutter run --release
 ```
 
-## Material motion for Flutter
+## Material Motion for Flutter
```

Comment:
Neither of the pages you've linked to capitalize "motion"; why are you doing so throughout (mostly) this PR?

Reply:
You're right, fixed